project/LibraryVersions.scala

```diff
@@ -3,7 +3,7 @@ object LibraryVersions {
   val circeVersion = "0.12.1"
   val awsClientVersion = "1.11.827"
   val catsVersion = "1.4.0"
-  val jacksonVersion = "2.10.5"
+  val jacksonVersion = "2.12.0"
```

Comment:
when I updated jackson, I stayed on the same minor version but just bumped the patch version.  That seemed to be enough to keep snyk happy but with less risk.

Reply:
see
https://github.com/guardian/members-data-api/blob/112029e534827aa476b450b7d8757ab0a9a2c9b9/project/Dependencies.scala#L47

Reply:
I see - jackson updates always seem pretty low risk to me! The tests pass and I've tested a contribution in CODE, so I think it's safe